src/main/java/com/liferay/ide/idea/ui/modules/LiferayModuleWizardStep.java

```diff
@@ -291,6 +292,13 @@ public boolean validate() throws ConfigurationException {
 
 		String type = getSelectedType();
 
+		if (type.equals("js-theme") || type.equals("js-widget")) {
```

Comment:
hey @simonjhy @Seiphon why does this not work in intellij?  Does this work in LiferayIDE?

Reply:
Hi @gamerson , LiferayIDE does not suported "js-theme" and "js-widget" modules for now, so I think Intellij doesn't suport them either, but I was wrong, these two modules works well on intellij. 
https://issues.liferay.com/browse/IDE-4795 This is the ticket which we need to add support to creating "js-theme" and "js-widget" on IDE, I will work on this ticket asap.

Reply:
Hi @gamerson 
These two modules are not supported by IDE or intellij yet. Because when create them, there are more parameters needed, as shown in the picture. we still work on how to better support these parameters.
![image](https://user-images.githubusercontent.com/24595644/96963264-f2292000-153a-11eb-9212-f2209d305b41.png)

Reply:
Thanks for update @Seiphon